src/style.css

```diff
@@ -1,3 +1,58 @@
-body {
+* {
   margin: 0;
 }
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  width: 1200px;
+  /*background-color: #444;*/
+}
+
+.nav_link {
+  display: inline-block;
+  height: 56px;
+  line-height: 56px;
+  /*background-color: #444;*/
+}
+
+.logo_img {
+  padding-left: 50px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 250px;
+  /*background-color: #444;*/
+}
+
+.nav {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-content: center;
+  justify-content: flex-end;
+}
+
+.is-active::after {
+  display: flex;
+  content: "";
+  height: 4px;
+  background-color: #00acdc;
+  color: #00acdc;
+  border-radius: 8px;
+}
+
+a {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+}
+
+a:hover {
+  color: #00acdc;
```

Comment:
Consider using variables for repeating colors

Reply:
done